bplist/bplist.py

```diff
@@ -271,3 +272,17 @@ def unplist(s):
     from Foundation import NSData, NSPropertyListSerialization
     d = NSData.dataWithBytes_length_(s, len(s))
     return NSPropertyListSerialization.propertyListWithData_options_format_error_(d, 0, None, None)
+
+if __name__ == "__main__":
+    import os
+    import sys
+    import json
+    file_path = sys.argv[1]
+
+    with open(file_path, "rb") as fp:
+        data = fp.read()
+
+    out = BPListReader(data).parse()
+
+    with open(file_path + ".json", "w") as fp:
+        json.dump(out, indent=4)
```

Comment:
Missing a new line in the end of the file.

Reply:
Thanks! A great feature would be to add nested decoding
Great library!